atlp-rwanda/tech-strikers-bn-frontend
# #175636781 setup git actions

#### What does this PR do?
- This PR configures Git actions workflow
#### Description of Task to be completed?
- GitHub Actions Workflow is configured as a CI service used to build and test the application.
#### How should this be manually tested?
- Run `git clone https://github.com/atlp-rwanda/tech-strikers-bn-frontend.git` to clone the repo
- Run `cd tech-strikers-bn-frontend`
- Run `git checkout ch-setup-git-actions-175636781`
- Push the branch and observe in actions if the workflow has passed successufully
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#175636781](https://www.pivotaltracker.com/story/show/175636781)
#### Screenshots (if appropriate)

![Screenshot from 2021-02-01 16-59-04](https://user-images.githubusercontent.com/69037813/106475911-2784c880-64af-11eb-990b-7ca041892099.png)

#### Questions:
